Composer/packages/client/src/components/ProjectTree/treeItem.tsx

```diff
@@ -173,16 +194,26 @@ const onRenderItem = (item: IOverflowSetItemProps) => {
           />
         )}
         {item.displayName}
+        {item.errorContent && (
```

Comment:
Are these are the error indicators inside the project tree.

@zhixzhan This might be useful for the work your doing

Reply:
@srinaath Sure, It helps. 
One more requirements from my side is a missing style.
<img width="917" alt="image" src="https://user-images.githubusercontent.com/49866537/96421852-e03d3980-1229-11eb-8b55-8f440f99b28c.png">

Reply:
Interesting. I won't make that change here, but if I could learn more about the specifics of this I'll make another PR for that. Are skills the only things that can be greyed out? How will the client know if a skill is broken/missing?

Reply:
Lets discuss this in the evenings status sync